### PR TITLE
Removes dirty steel tiles

### LINKED ISSUE #20575: Dirty steel tiles should be removed
## Description
Considering we have dirt decals, there is zero reason at all to have these. I would have removed these in the tile resprite if I had known we could remove tiles beforehand.
They're also frequently used to stack dirt decals on top of eachother (which looks really bad.)

### PR DESCRIPTION
**Pull request by Flareguy**
## About the PR
Removes dirty steel tiles from the game, and replaces them with regular ones.

Fixes #20575 
## Why / Balance
I planned to do this back during the tile resprite, but wasn't aware that tile migrations existed so I left them in. However, as you can tell, they do in fact exist.

Dirty steel tiles have no reason to exist, plain-and-simple. You can just use dirt decals in place of them for the same effect, with the addition of them being cleanable as well. While they *do* function as baked-in grime, they're barely different looking then the dirt decals and literally use the same base.
They also function as a noob trap - Core, for instance, after this change, is squeaky clean - there isn't a single dirt decal in sight. It also occasionally encourages people to stack them with dirt decals, which looks horrible.

I haven't checked the commit logs, but I would be willing to bet dirty steel tiles predate the existence of decals, especially considering they used Eris(?) sprites before moony's tile resprite changed them.

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
Basically every map in the game uses these as far as I know, mappers will almost certainly need to re-dirt:tm: their areas after this change. Don't forget to set them to cleanable!

**Changelog**

:cl:
- remove: Removed dirty steel tiles (dirt decals still exist.)

### CODE REVIEW

**Comment by brainfood1183** [comment]
isnt it actually a rusty steel tile and just named wrong?

**Comment by IamVelcroboy** [comment]
> isnt it actually a rusty steel tile and just named wrong?

I agree with this. I still don't like removing. These are nice to have for certain areas where you want a permanently stained tile. "Rust" is actually a better concept/description. Setting decals to not be cleanable only sort of works as decals are removed when tiles are removed/replaced.

**Comment by Carolyn3114** [comment]
> isnt it actually a rusty steel tile and just named wrong?

there's a rust decal

**Comment by EmoGarbage404** [comment]
i would rather just see the dirty steel tile resprited to be more distinct than steel + dirt decal. Having a "caked on" dirty tile is useful for various applications.

**Comment by UbaserB** [comment]
As a mapper, this is cruel to remove one of my favourite tiles

**Comment by Partmedia** [comment]
I'd like to add that it's not great for forks to be forced to remove this tile. At best these tiles should be deprecated from new future upstream maps. Unless you're willing to "re-dirt™️" all of the existing uses of this tile upstream, this tile should stay.

**Comment by brainfood1183** [comment]
> i would rather just see the dirty steel tile resprited to be more distinct than steel + dirt decal. Having a "caked on" dirty tile is useful for various applications.

i agree with this, keep the Id just rename tile and resprite it to something like a new steel maints tile.

**Comment by Cheackraze** [comment]
Ye don't remove the tile completely, I definitely still use it in mapping, there's no need to remove it

**Comment by Partmedia** [comment]
Closing per the discussion. It's important to consider forks as well as extra mapper effort to fix any maps using this.
